IPLD-Path.md

```diff
@@ -0,0 +1,20 @@
+# [WIP] IPLD Path v1
+
+An IPLD Path is a string identifier used for deep references into IPLD
+graphs.
+
+IPLD Path's are constructed following the same constraints as [URI Paths](https://tools.ietf.org/html/rfc3986#section-3.3).
```

Comment:
Does it imply that `#` is also reserved?

Reply:
Yes. The fact is, if we don't go "all in" on URI path semantics we can't use existing URI path parsers and escape libraries. I also think that any deviation we make will come back to bite us later when we do more browser integration.

Also, you can still have all of these special characters in keys that are referenced by paths as long as you escape them properly.